tool/teleport/common/teleport.go

```diff
@@ -305,6 +305,26 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dump.Flag("app-uri", "Internal address of the application to proxy.").StringVar(&dumpFlags.AppURI)
 	dump.Flag("node-labels", "Comma-separated list of labels to add to newly created nodes, for example env=staging,cloud=aws.").StringVar(&dumpFlags.NodeLabels)
 
+	dumpNode := app.Command("node", "SSH Node configuration commands")
+	dumpNodeConfigure := dumpNode.Command("configure", "Generate a configuration file for an ssh node.")
+	dumpNodeConfigure.Flag("cluster-name",
+		"Unique cluster name, e.g. example.com.").StringVar(&dumpFlags.ClusterName)
+	dumpNodeConfigure.Flag("output",
+		"Write to stdout with -o=stdout, default config file with -o=file or custom path with -o=file:///path").Short('o').Default(
+		teleport.SchemeStdout).StringVar(&dumpFlags.output)
+	dumpNodeConfigure.Flag("acme",
+		"Get automatic certificate from Letsencrypt.org using ACME.").BoolVar(&dumpFlags.ACMEEnabled)
+	dumpNodeConfigure.Flag("acme-email",
+		"Email to receive updates from Letsencrypt.org.").StringVar(&dumpFlags.ACMEEmail)
+	dumpNodeConfigure.Flag("version", "Teleport configuration version.").Default(defaults.TeleportConfigVersionV2).StringVar(&dumpFlags.Version)
+	dumpNodeConfigure.Flag("public-addr", "The hostport that the proxy advertises for the HTTP endpoint.").StringVar(&dumpFlags.PublicAddr)
+	dumpNodeConfigure.Flag("cert-file", "Path to a TLS certificate file for the proxy.").ExistingFileVar(&dumpFlags.CertFile)
+	dumpNodeConfigure.Flag("key-file", "Path to a TLS key file for the proxy.").ExistingFileVar(&dumpFlags.KeyFile)
+	dumpNodeConfigure.Flag("data-dir", "Path to a directory where Teleport keep its data.").Default(defaults.DataDir).StringVar(&dumpFlags.DataDir)
+	dumpNodeConfigure.Flag("token", "Invitation token to register with an auth server.").StringVar(&dumpFlags.AuthToken)
+	dumpNodeConfigure.Flag("auth-server", "Address of the auth server.").StringVar(&dumpFlags.AuthServer)
+	dumpNodeConfigure.Flag("node-labels", "Comma-separated list of labels to add to newly created nodes ex) env:staging,cloud:aws.").StringVar(&dumpFlags.NodeLabels)
```

Comment:
I think this was pointed out before, but other commands use `(e.g. key1=value1,key2=value2)` for providing labels. I think having two different ways to do the same thing will confuse people Can we be consistent here?

Reply:
oops good catch, bad merge.